### PR TITLE
Enable scanning

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

**Review by mobileoverlord — APPROVED** [review]
This looks good

**Comment by mobileoverlord** [comment]
Sorry for the delay, bee catching up still. Could you add to the readme with a short blurb regarding the experience of scanning for wifi networks :)

**Comment by ConnorRigby** [comment]
@mobileoverlord ive been using it for a while, and it hangs in an infinite loop inside `nerves_wpa_supplicant`. I opened a pull request there too. 

Yeah ill take a quick look at the the readme

**Comment by mobileoverlord** [comment]
ah, looks like the receive loop there needs to break

File: lib/interim_wifi/if_supervisor.ex [hunk 1]
```diff
@@ -36,6 +36,25 @@ defmodule Nerves.InterimWiFi.IFSupervisor do
...
+       GenServer.call(pidname, :scan, 30_000)
+     else
+       {:error, :not_started}
+     end
+  end
+
+  def status(ifname) do
+     pidname = pname(ifname)
+     if Process.whereis(pidname) do
+       # im not sure what status is needed here?
```
Comment (GregMefford) [inline lib/interim_wifi/if_supervisor.ex]: @mobileoverlord Do you know what's the correct thing to have here? I'd like to remove this comment if this is the right thing to do here.

**Review by GregMefford — CHANGES_REQUESTED** [review]
Just one small item to address, but otherwise LGTM! 👍

File: lib/interim_wifi/if_supervisor.ex (hunk 1 above)
Comment (ConnorRigby) [reply lib/interim_wifi/if_supervisor.ex]: Ah yeah i forgot that was in there. I meant to ask @fhunleth about that. There's two status functions one in `Nerves.NetworkInterface`, and one in `Nerves.WpaSupplicant`

**Review by mattneel — COMMENTED** [review]
Should be:
```
System.cmd("modprobe", ["brcmfmac"])
```

Ah, you got it. Nevermind.

File: lib/interim_wifi/if_supervisor.ex (hunk 1 above)
Comment (mobileoverlord) [reply lib/interim_wifi/if_supervisor.ex]: I think for now getting the NetworkInterface status is best since not every interface will use WpaSupplicant

**Review by GregMefford — APPROVED** [review]
👍

**Pull request merged**